docs_source_files/content/introduction/the_selenium_project_and_tools.es.md

```diff
@@ -3,121 +3,45 @@ title: "El proyecto Selenium y herramientas"
-for a niché audience and a specific use case,
-to the world's _de facto_ browser automation library.
-
-Just as Selenium RC made use of the tools of the trade available at the time,
-[Selenium WebDriver]({{< ref "/webdriver/_index.md" >}}) drives that tradition on by taking
-the browser interaction part to the browser vendor's home turf,
-and asking them to take responsibility of the backend, browser-facing implementations.
-Recently this work has evolved into a W3C standardisation process
-where the goal is to turn the WebDriver component in Selenium
-into the _du jeur_ remote control library for user agents.
\ No newline at end of file
+### Selenium controlador de navegadores web
+
+_Selenium_ es muchas cosas, pero en esencia es un conjunto de herramientas para la automatización del navegador web que utiliza las mejores técnicas disponibles para controlar de forma remota las instancias del navegador y emular la interacción de un usuario con el navegador.
+
+Permite a los _testers_ simular actividades comunes realizadas por los usuarios finales; ingresar texto en los campos, seleccionar valores desplegables y casillas de verificación, y hacer clic en los enlaces de los documentos. También proporciona muchos otros controles, como el movimiento del mouse, la ejecución arbitraria de JavaScript y mucho más.
+
+Aunque se utiliza principalmente para la prueba de front-end de sitios web, Selenium es en esencia una _Librería_ de agente de usuario de navegador. Las interfaces son ubicuas a la aplicación, lo que fomenta la relación con otras librerías para adaptarse a su necesidad.
+
+
+### Una interfaz para gobernarlos a todos
+
+Uno de los principios rectores del proyecto es admitir una interfaz común para todas las tecnologías de navegador (principales).
+Los navegadores web son aplicaciones increíblemente complejas y de alta ingeniería, que realizan sus operaciones de formas completamente diferentes, pero que con frecuencia se ven iguales al hacerlo. Aunque el texto se representa en las mismas fuentes, las imágenes se muestran en el mismo lugar y los enlaces lo llevan al mismo destino. Lo que sucede debajo es tan diferente como la noche y el día.
+Selenium "resume" estas diferencias, ocultando sus detalles y complejidades a la persona que escribe el código. Esto le permite escribir varias líneas de código para realizar un flujo de trabajo complicado, pero estas mismas líneas se ejecutarán en Firefox, Internet Explorer, Chrome y todos los demás navegadores compatibles.
+
+
+### Herramientas y soporte
+
+El enfoque de diseño minimalista de Selenium le da versatilidad para ser incluido como componente en aplicaciones más grandes. La infraestructura circundante proporcionada bajo Selenium le brinda las herramientas para armar su propia [cuadrícula de navegadores]({{< ref "/grid/_index.md" >}}) o _grid_ para que las pruebas se puedan ejecutar en diferentes navegadores y múltiples sistemas operativos en una amplia gama de máquinas.
```

Comment:
`cuadrícula de navegadores`, let's just keep it simple and use the word `Grid` for it.

Reply:
Solved